go/src/socialapi/workers/api/modules/message/controller.go

```diff
@@ -255,6 +255,10 @@ func Delete(u *url.URL, h http.Header, _ interface{}, c *models.Context) (int, h
 
 		// delete the message here
 		err = cm.DeleteMessageAndDependencies(false)
+		if err != nil {
+			return response.NewBadRequest(err)
```

Comment:
I don't think it's correct - it changes the flow: previously if deleting message and deps failed, we still invalidated the cache. Right not we do not.